kanopi/kanopi-components
# Array Manipulation Methods & Initial Asset Loader Configuration Parsing

## Description
Teamwork Ticket(s): [Asset Loader Rebuild](https://kanopi.teamwork.com/app/tasks/29110580)

Primarily this introduces some common Array/Iterator interfaces and methods to the Arrays transform to provide common array manipulations in a chainable, fluent fashion.

This is an initial preview release of the port/rebuild of Kanopi Pack asset loader methods into the Components library 

## Steps to Validate
1. Pull the branch
2. Run `make` assuming XCode tools are installed, all tests and PHPCS sniffs should pass
